HashRegistrarSimplified.sol

```diff
@@ -231,6 +295,11 @@ contract Registrar {
         bid.setOwner(_owner);
         entry h = entries[_hash];
 
+        /* 
+        A penalty is applied for submitting pointless bids. This prevents bids which
+        are never intended to be revealed during the reveal period, which could otherwise
+        appear falsely as an expression of interest.
```

Comment:
No; suppose I win the auction for 10 ether, and the next bid is 5 ether; I would expect to pay 5 ether. If there's no penalty for failing to reveal bids, Susan could place a bid every 1 ether increment, then after the auction come along and say "I will reveal this bid for 9 ether unless you pay me 1 ether not to".